java/build/pom.xml

```diff
@@ -103,5 +103,11 @@
             <artifactId>json-unit-assertj</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <version>3.4.1</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
This is a duplicate of AssertJ's dependency entry in the root pom, except that it overrides the managed version number to an earlier version of the library. It looks like this can be removed?